### ISSUE TITLE
Statically allocate string concatinations using FMT

### ISSUE THREAD

**Issue by saikishor** [ros-controls/ros2_control#1906]
could we please add a follow-up issue to make this statically allocated by using FMT?

_Originally posted by @bmagyar in https://github.com/ros-controls/ros2_control/pull/1871#discussion_r1866454408_

**Comment by mini-1235** [comment]
I am not sure if I understand this issue correctly, does it mean to add something like `fmt::format` to replace the original `const std::string`?

**Comment by christophfroehlich** [comment]
I think so, yes @saikishor?

**Comment by saikishor** [comment]
@mini-1235 Yes, we need to do it similar way, the only thing is it is missing `FMT_COMPILE` . You can check more details here : https://fmt.dev/9.1.0/api.html#format-string-compilation

**Comment by mini-1235** [comment]
> [@mini-1235](https://github.com/mini-1235) Yes, we need to do it similar way, the only thing is it is missing `FMT_COMPILE` . You can check more details here : https://fmt.dev/9.1.0/api.html#format-string-compilation

@saikishor thanks for the guidance! I have opened a PR for it
